MergeSharpWebAPI/Hubs/Clients/IFrontEndClient.cs

```diff
@@ -5,6 +5,6 @@ namespace MergeSharpWebAPI.Hubs.Clients;
 public interface IFrontEndClient
 {
     Task ReceiveMessage(LWWSetModel<int> message);
-    Task ReceiveMessage(TPTPGraphModel message);
+    Task ReceiveMessage(List<Node> message);
```

Comment:
would be good to add a `TODO:` here as well. I assume this shouldn't be a List<Node>. 

Ok to leave for now though because we won't be using this code.

Reply:
O I see, should it be like an Enumerable??

Reply:
nah shouldnt it be the Json object of nodes and edges? This is the object the frontend expects to receive right?

Reply:
Ohhh yessir u rite